packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -142,10 +352,33 @@ export default class Dropdown extends React.Component<DropdownProps> {
                     styles.dropdown,
                     light && styles.light,
                     outOfBoundaries && styles.hidden,
+                    {minWidth: minDropdownWidth},
                     dropdownStyle,
                 ]}
             >
-                {children}
+                {items.map((item, index) => {
+                    if (item.component.type === SeparatorItem) {
+                        return item.component;
+                    } else {
+                        // if (index === 0) {console.log(item);}
+                        if (item.focusable) {
+                            focusCounter += 1;
+                        }
+                        const focusIndex = focusCounter - 1;
+                        return React.cloneElement(item.component, {
+                            ...item.populatedProps,
+                            key: index,
+                            ref: item.focusable && this.getRef(index),
+                            onClick: () => {
```

Comment:
This `onClick` overwrite the `onClick` from `item.populatedProps`.

Reply:
`populatedProps` no longer tries to set an `onClick`